config/config.exs

```diff
@@ -0,0 +1,68 @@
+# This file is responsible for configuring your application
+# and its dependencies with the aid of the Mix.Config module.
+#
+# This configuration file is loaded before any dependency and
+# is restricted to this project.
+use Mix.Config
+
+# Customize non-Elixir parts of the firmware. See
+# https://hexdocs.pm/nerves/advanced-configuration.html for details.
+
+config :nerves, :firmware, rootfs_overlay: "rootfs_overlay"
+
+# Use shoehorn to start the main application. See the shoehorn
+# docs for separating out critical OTP applications such as those
+# involved with firmware updates.
+
+config :shoehorn,
+  init: [:nerves_runtime, :nerves_init_gadget],
+  app: Mix.Project.config()[:app]
+
+# Use Ringlogger as the logger backend and remove :console.
+# See https://hexdocs.pm/ring_logger/readme.html for more information on
+# configuring ring_logger.
+
+config :logger, backends: [RingLogger]
+
+# Authorize the device to receive firmware using your public key.
+# See https://hexdocs.pm/nerves_firmware_ssh/readme.html for more information
+# on configuring nerves_firmware_ssh.
+
+keys =
+  [
+    Path.join([System.user_home!(), ".ssh", "id_rsa.pub"]),
+    Path.join([System.user_home!(), ".ssh", "id_ecdsa.pub"]),
+    Path.join([System.user_home!(), ".ssh", "id_ed25519.pub"])
+  ]
```

Comment:
This is probably auto generated by Nerves, right? Welp, who cares, I'm gonna review this anyway.

---

You could simplify this:

```elixir
keys =
  [
    "id_rsa.pub",
    "id_ecdsa.pub",
    "id_ed25519.pub"
  ]
  |> Enum.map(&Path.join([System.user_home!(), ".ssh", &1]))
  |> Enum.filter(&File.exists?/1)
```

Reply:
Yeah, this is is nerves default stuff. Guess you could PR it on nerves_init_gadget. Though I think it might be more clear (less clever the way it is :)